website/docs/getting-started/pushing-an-image-to-a-registry.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 10
+title: Pushing an image to a registry
+description: Pushing an image to a registry.
+keywords: [podman desktop, podman, containers, image, registry, registries]
+tags: [images]
+---
+
+# Pushing an image to a registry
+
+With Podman Desktop, you can push an image to registries.
+
+#### Prerequisites
+
+- You have configured your registry **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**.
+- You have built an image, which name is the fully qualified name required for your registry, such as `quay.io/my-repository/my-image`, `ghcr.io/my-repository/my-image`, or `docker.io/my-repository/my-image`.
```

Comment:
should we say that image need to have the "public name" (be prefixed by the registry)

I mean, if the image is named 'foo', I can't push it to `quay.io/my-repo/foo`, name should be `quay.io/my-repo/foo`

Reply:
I mean, it's not obvious on how it's written now.

Reply:
How would you name it? "fully qualified name" is not correct?